lib/turn.rb

```diff
@@ -19,7 +19,7 @@ def get_move
 
   def validate_move(move, valid_columns)
     while !valid_columns.include?(move) || column_is_full?(move, valid_columns)
-      puts "Oops, that's an invalid move. Please select column A-G!"
+      puts "Oops, that's an invalid move, or a full column. Please select column A-G!"
```

Comment:
I would just leave it as column maybe? not really a bit deal, easily changeable.